### ISSUE TITLE
Video controls are not visible in high contrast white mode.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1378]
**Actual Result:**
Video controls are not visible in high contrast white mode.

**Note:**
Text of 'Watch later' and 'Share' button of YouTube video is not visible in high contrast white mode

**Expected Result:**
Video controls should be visible in high contrast white mode.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Navigate to 'Show a video attachment from Vimeo' button and activate it.
3. Video will appear at the bottom of the page.
4. Go to Settings->Ease of access->Apply High Contrast White mode.
5. Observe if video controls are visible or not.

**User Impact:**
If any image , icon , hyperlink , text or visual focus disappear when high contrast mode is enabled , will make the UI confusing and difficult for users with low vision to navigate easily to the desired element.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**MAS Reference:**
[MAS 4.3.1 - No Disruption of Accessibility Features (42A)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74})

**Attachment:**
[MAS4.3.1_HighContrast.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2598837/MAS4.3.1_HighContrast.zip)

**Comment by compulim** [comment]
This is not a bug from Web Chat. The video content is an IFRAME owned by Vimeo.

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with Accessibility team as External

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.3.1
